Add source name to generated pull request's title

chrismytton opened the pull request:
This adds the name of the source that was rebuilt to the pull request title if a rebuild for a single source has been requested.

# Notes to reviewer

I tried adding some tests for this but didn't get very far because of the complexity of the `RebuilderJob#perform` method. I've opened #62 to track refactoring that, but we think that getting this fix in still adds value, even if it isn't tested.

Fixes https://github.com/everypolitician/rebuilder/issues/59

tmtmtmtm commented on app.rb:
```diff
@@ -76,7 +76,7 @@ def perform(country_slug, legislature_slug, source = nil)
     output = output[-64_000..-1] || output
-    title = "#{country.name} (#{legislature.name}): refresh data"
+    title = "#{country.name} (#{legislature.name}#{source ? ":#{source}" : ''}): refresh data"
     body = "Automated data refresh for #{country.name} - #{legislature.name}" \
```
I think we’ll want to show this in the body text immediately below as well, so it’s probably better to pull the country/legislature/source combo into a variable and then interpolate it into both title and body (I don't think it matters that it's a slight change to the format we use in the body)

tmtmtmtm replied:
The text here is slightly odd too. This will give us, for example "South Korea (National Assembly:wikidata): refresh data"

After some discussion on slack, we've decided on:

"South Korea (National Assembly): refresh wikidata"

and the body:

"Automated refresh of Wikidata for South Korea - National Assembly"

with, in both cases, a fallback of "all data" if there's no source.

chrismytton committed:
Add source name to pull request title

If a rebuild is for a specific source then include that in the title and
the body of the pull request. Otherwise it will say "all sources".

chrismytton commented:
You can see this in action for a single source - https://github.com/chrismytton/everypolitician-data/pull/17, and for all sources - https://github.com/chrismytton/everypolitician-data/pull/18.

tmtmtmtm reviewed (approved):
Yep, I agree.
